colibre/config.yml

```diff
@@ -508,3 +508,8 @@ scripts:
     caption: SNIa rate history plotted directly from the SNIa.txt file produced by SWIFT.
     section: SNIa Rate
     output_file: SNIa_rate_history.png
+  - filename: scripts/dust_mass_evolution.py
+    caption: Evolution of the dust mass density plotted directly from the statistics.txt file produced by SWIFT.
+    output_file: dust_mass_evolution.png
+    section: Star Formation History
```

Comment:
Is this the correct section that we want for this plot?

Reply:
I thought next to the SF and stellar mass density was good.

Reply:
Feel free to change it to something you prefer.